locale/en_US/editor.po

```diff
@@ -368,6 +368,9 @@ msgstr "Last Complete"
 msgid "editor.submissions.assignedTo"
 msgstr "Assigned To"
 
+msgid "issues.submissions.issueIds"
+msgstr "Assigned Issues"
```

Comment:
Let's call this `Assigned to Issue`. Let's also change `Assigned To` for the editor filter to `Assigned to Editor`.